server.ts

```diff
@@ -166,6 +170,21 @@ const getRepoInfo = async (res: any, groupBy: string) => {
   res.status(200).json({ nodes, edges, filePaths });
 };
 
+const getRepoReadme = async (res: any) => {
+  const files = await getFilesInDirectory(REPO_PATH);
+  if(!files) {
+    res.status(404).json({ error: { message: "Repository not cloned yet" } });;
+    return;
+  }
+  const readmePth = files.find(f => f.toLowerCase().includes('readme.md'));
```

Comment:
Do we have to `toLowerCase()`? Can't we expect all users to call it `README.md`? Or is that more of a convention rather than a rule? (like would github pick up on a `readme.md` instead of `README.md`?)